models/Asset.php

```diff
@@ -1308,7 +1308,15 @@ public function setStream($stream)
         if (is_resource($stream)) {
             $this->setDataChanged(true);
             $this->stream = $stream;
-            rewind($this->stream);
+
+            $isRewindable = @rewind($this->stream);
```

Comment:
```php
  $meta = stream_get_meta_data($stream);
  $seekable = $meta['seekable'];
```

Reply:
Hmm, could also be done this way, but would be an additional function call because for local (rewindable) streams, the stream has to to rewinded.

Reply:
I'm happy with `@rewind()` if it doesn't have any other implications.

Reply:
@BlackbitNeueMedien One more thing, ... the temp file should be cleaned up, once the asset object get's destroyed ... just add the following line after your code and it should be fine :) 

```php
$this->_temporaryFiles[] = $tmpFile;
```

Thanks!

Reply:
@BlackbitNeueMedien Maybe also rebase/merge to the current master, so that the tests can complete without any errors :) Thanks again!